### PR TITLE
Fix multiple issues in custom-elements/custom-elements-registry/define.html

### PR DESCRIPTION
**Pull request by kojiishi**
- Names should not use uppercase, or it will throw SyntaxError.
- 8. observedAttributes rethrow test should use static method.
- 12-17. Add more types to test IsCallable for callbacks.

### CODE REVIEW

File: custom-elements/custom-elements-registry/define.html [hunk 1]
```diff
@@ -208,42 +215,38 @@
...
     'connectedCallback',
     'disconnectedCallback',
     'attributeChangedCallback',
-  ].forEach(name =>  {
-    test(() =>  {
+  ].forEach(name => {
+    test(() => {
       class C {
-        get [name]() { throw_rethrown_error(); }
+        get[name]() { throw_rethrown_error(); }
```
Comment (domenic) [inline custom-elements/custom-elements-registry/define.html]: Nit: I'd keep this space. For the non-computed case, `get observedAttributes() { }`, it's necessary, so it's nice to be consistent.

File: custom-elements/custom-elements-registry/define.html [hunk 2]
```diff
@@ -208,42 +215,38 @@
...
+        class C { };
+        C.prototype[name] = data.value;
+        if (data.success) {
+          customElements.define(`test-define-${name.toLowerCase()}-${data.name}`, C);
+        } else {
+          assert_throws(expectTypeError, () => {
+            customElements.define(`test-define-${name.toLowerCase()}-${data.name}`, C);
+          });
+        }
+      }, `If constructor.prototype.${name} is ${data.name}, should ${data.success ? 'success' : 'throw a TypeError'}`);
```
Comment (domenic) [inline custom-elements/custom-elements-registry/define.html]: Nit: the string in the message should be "should succeed", not "should success"

**Comment by domenic** [comment]
LGTM

File: custom-elements/custom-elements-registry/define.html (hunk 1 above)
Comment (kojiishi) [inline custom-elements/custom-elements-registry/define.html]: Done.

File: custom-elements/custom-elements-registry/define.html (hunk 2 above)
Comment (kojiishi) [inline custom-elements/custom-elements-registry/define.html]: Done.

**Pull request merged**